test/account/UpgradeableModularAccount.t.sol

```diff
@@ -402,17 +402,15 @@ contract UpgradeableModularAccountTest is AccountTestBase {
     function test_isValidSignature() public {
         bytes32 message = keccak256("hello world");
 
+        bytes32 replaySafeHash = vm.envOr("SMA_TEST", false)
+            ? SemiModularAccount(payable(account1)).replaySafeHash(message)
+            : singleSignerValidationModule.replaySafeHash(address(account1), message);
+
         uint8 v;
         bytes32 r;
         bytes32 s;
 
-        if (vm.envOr("SMA_TEST", false)) {
-            // todo: implement replay-safe hashing for SMA
-            (v, r, s) = vm.sign(owner1Key, message);
-        } else {
-            bytes32 replaySafeHash = singleSignerValidationModule.replaySafeHash(address(account1), message);
-            (v, r, s) = vm.sign(owner1Key, replaySafeHash);
-        }
+        (v, r, s) = vm.sign(owner1Key, replaySafeHash);
```

Comment:
Style nit: can inline declare vars now as:
```solidity
(uint8 v, bytes32 r, bytes32 s)
```

Reply:
Done in [df43ba7](https://github.com/erc6900/reference-implementation/pull/155/commits/df43ba75580eb7403bb9fd553d74f5c6463f6d05)!